docs/zh-CN/docs/lakehouse/multi-catalog/jdbc.md

```diff
@@ -66,23 +67,113 @@ PROPERTIES ("key"="value", ...)
 
 3. Http 地址。如：`https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/jdbc_driver/mysql-connector-java-8.0.25.jar`。系统会从这个 http 地址下载 Driver 文件。仅支持无认证的 http 服务。
 
-### 小写表名同步
+### 小写名称同步
 
-当 `lower_case_table_names` 设置为 `true` 时，Doris 通过维护小写名称到远程系统中实际名称的映射，能够查询非小写的数据库和表以及列
+当 `lower_case_meta_names` 设置为 `true` 时，Doris 通过维护小写名称到远程系统中实际名称的映射，使查询时能够使用小写去查询外部数据源非小写的数据库和表以及列。
+
+由于 FE 存在 lower_case_table_names 的参数，会影响查询时的表名大小写规则，所以规则如下
+
+* 当 FE lower_case_table_names config 为 0 时
+
+   lower_case_meta_names = false，大小写和源库一致。
+   lower_case_meta_names = true，小写存储库表列名。
+
+* 当 FE lower_case_table_names config 为 1 时
+
+   lower_case_meta_names = false，db 和 column 的大小写和源库一致，但是 table 存储为小写
+   lower_case_meta_names = true，小写存储库表列名。
+
+* 当 FE lower_case_table_names config 为 2 时
+
+   lower_case_meta_names = false，大小写和源库一致。
+   lower_case_meta_names = true，小写存储库表列名。
+
+如果创建 Catalog 时的参数配置匹配到了上述规则中的转变小写规则，则 Doris 会将对应的名称转变为小写存储在 Doris 中，查询时需使用 Doris 显示的小写名称去查询。
+
+如果外部数据源存在名称相同只有大小写不同的情况，例如 DORIS 和 doris，Doris 由于歧义而在查询 Catalog 时报错，此时需要配置 `suffix_names_mapping` 参数来解决冲突。
+
+`suffix_names_mapping` 参数接受一个 Json 格式的字符串，格式如下：
+
+```json
+{
+  "databases": [
+    {
+      "remoteDatabase": "DORIS",
+      "mapping": "doris_1"
+    },
+    {
+      "remoteDatabase": "doris",
+      "mapping": "doris_2"
+    }],
+  "tables": [
+    {
+      "remoteDatabase": "DORIS",
+      "remoteTable": "DORIS",
+      "mapping": "doris_1"
+    },
+    {
+      "remoteDatabase": "DORIS",
+      "remoteTable": "doris",
+      "mapping": "doris_2"
+    }],
+  "columns": [
+    {
+      "remoteDatabase": "DORIS",
+      "remoteTable": "DORIS",
+      "remoteColumn": "DORIS",
+      "mapping": "doris_1"
+    },
+    {
+      "remoteDatabase": "DORIS",
+      "remoteTable": "DORIS",
+      "remoteColumn": "doris",
+      "mapping": "doris_2"
+    }]
+}
+```
+
+在将此配置填写到创建 Catalog 的语句中时，需要对双引号加上转义符，例如：
```

Comment:
Can use single quote? like `'{"xx":"yy"}'`

Reply:
Two methods are re-explained in the document, single quotes and double quotes